sources/packages/backend/apps/api/src/main.ts

```diff
@@ -87,6 +88,12 @@ async function bootstrap() {
       .build();
     const document = SwaggerModule.createDocument(app, options);
     SwaggerModule.setup("swagger", app, document);
+
+    // External swagger
+    const externalDocument = SwaggerModule.createDocument(app, options, {
+      include: [AppExternalModule], // Includes only AppExternalModule.
+    });
+    SwaggerModule.setup("external/swagger", app, externalDocument);
```

Comment:
We need to ensure this URL will be configured in the deployment config.

Reply:
Yes, sorry. I forgot that. I'm going to add it.